src/sinks/util/unix.rs

```diff
@@ -71,14 +90,20 @@ impl UnixSinkConfig {
     }
 }
 
+enum UnixEither {
```

Comment:
I named it this way so that it matched `sinks::util::service::net::unix::UnixEither`. Initially, I wanted to import it, but I would have to change the visibility of this enum, which didn't feel like the right thing to do.
Given this context, should I still rename it per your suggestion?

Reply:
Good point 👍 We should also move that one and its `impl` here since the enums are identical. We can keep the `UnixEither` name.

Reply:
Done in f30e104aef0e7b63b479b02cd8edfb25adda0e07.